README.md

```diff
@@ -64,9 +107,8 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Thanks to my family for all the support that thay give me.
+- [Original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) created by [minimmmal](https://www.behance.net/gregoirevella) in Behance.
```

Comment:
- [ ] In this acknowledgement, please give credit to the [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). I can see you did credit Behance, but the original author was left out.